Basicfilter.js

```diff
@@ -0,0 +1,6 @@
+function getShortMessages(messages) {
+    // SOLUTION GOES HERE
```

Comment:
Remember to remove unnecessary code. Now that you already have a solution, this comment is not needed, don't you think?